jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/SkaffoldInitOutput.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.plugins.common;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import javax.annotation.Nullable;
+
+/**
+ * Builds a JSON string containing the configured target image and sub-project name, to be consumed
+ * by <a href="https://github.com/GoogleContainerTools/skaffold">Skaffold</a>.
+ *
+ * <p>Example:
+ *
+ * <pre>{@code
+ * {
+ *   "image":"gcr.io/project/test",
+ *   "project":"project-name"
+ * }
+ * }</pre>
+ */
+public class SkaffoldInitOutput {
+
+  @JsonInclude(Include.NON_NULL)
+  @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+  private static class SkaffoldInitTemplate {
+
+    @Nullable private String image;
+
+    @Nullable private String project;
```

Comment:
I know this is the same pattern in `SkaffoldFilesOutput`, but I wonder if `SkaffoldInitOutput` can have these fields directly. Why do we have to declare another class for the additional wrapping? Is jackson not able to take care of it?